docs/root/intro/arch_overview/security/rbac_filter.rst

```diff
@@ -83,6 +83,7 @@ The following attributes are exposed to the language runtime:
    response.headers, string map, All response headers
    response.trailers, string map, All response trailers
    response.size, int, Size of the response body
+   response.total_size, int, Total size of the response including the headers and the trailers
```

Comment:
Unpacked or on the wire (i.e. does it account for HPACK/QPACK, chunked encoding, etc.)?

Reply:
I had the same question. I'm using bytes() property of the header map. I think it's reporting the compressed size, since the numbers don't seem to add up.

Reply:
Hmm, I thought that `bytes()` is used to make sure that uncompressed headers don't exceed the configured limit, so that shouldn't be the case.

@asraa you're probably the authority here, any ideas?

Reply:
bytes() is approximate uncompressed size.

Reply:
FWIW existing code uses the same method to get the sizes: https://github.com/istio/proxy/blob/master/src/envoy/http/mixer/filter.cc#L61. So far noone complained that it's not precise.

Reply:
This looks fine to me in general, so will defer to all of you if you want different doc text or you want to go with this.

Reply:
I think it would be helpful to clarify that this is "approximate uncompressed size of headers and trailers" and not the wire size.

Reply:
Sure, updated the doc.